index.js

```diff
@@ -61,12 +61,13 @@ function mnemonicToSeedHex (mnemonic, password) {
 function mnemonicToEntropy (mnemonic, wordlist) {
   wordlist = wordlist || DEFAULT_WORDLIST
 
-  var words = unorm.nfkd(mnemonic).split(' ')
+  var words = mnemonic.split(wordlist.separator)
```

Comment:
@fanatid was `unorm` being used here to normalize the separator?  I don't think we need it now?

Reply:
@dcousens This is not good. There is a possibility the user will input using normal spaces. (depending on which device/keyboard/IME they use)

normalizing beforehand fixed this, since both ASCII space and \u3000 space would both turn into ASCII space.

Your new method will fail if a Japanese user uses ASCII spaces for whatever reason.

Also, Japanese fails on new line 65 `var index = wordlist.words.indexOf(word)` because the wordlist is normalized strings but normal Japanese input is different String-wise (and Buffer-wise).

The normalize needs to stay, unfortunately... :-/

Reply:
If the normalized separator = ASCII space assumption is ever broken for a new wordlist we'd need to revisit this... maybe it needs a comment.

Also, the `var index = wordlist.words.indexOf(word)` actually is a "every language except English" problem, and not Japanese specifically.

Reply:
sounds like there needs to be a test for at least a japanese seed phrase